.reek

```diff
@@ -34,4 +34,4 @@ UtilityFunction:
     - SmsSenderConfirmationJob
 exclude_paths:
   - db/migrate
-  - spec/**/*
+  - spec
```

Comment:
In the long run I'm not totally sure about this.  I think test code should smell good too.. maybe we should relax some of the rules only?